ts/src/kucoinfutures.ts

```diff
@@ -1760,14 +1762,14 @@ export default class kucoinfutures extends kucoin {
             'timestamp': undefined,
             'datetime': undefined,
             'fundingRate': this.safeNumber (data, 'predictedValue'),
-            'fundingTimestamp': undefined,
-            'fundingDatetime': undefined,
+            'fundingTimestamp': fundingTimestamp,
```

Comment:
hum this timestamp seems to be the "nextFundingTimestamp`  (right now are 14h29 UTC time and the next funding it at 20h00)
![image](https://github.com/ccxt/ccxt/assets/43336371/f6492cbf-00b5-459c-b61a-b97b6e46bed1)


```
CCXT v4.0.57
kucoinfutures.fetchFundingRate(ETH/USDT:USDT)
{'datetime': None,
 'estimatedSettlePrice': None,
 'fundingDatetime': '2023-08-12T20:00:00.000Z',
 'fundingRate': 0.0001,
 'fundingTimestamp': 1691870400000,
 'indexPrice': None,
 'info': {'granularity': 28800000,
          'predictedValue': 0.0001,
          'symbol': '.ETHUSDTMFPI8H',
          'timePoint': 1691841600000,
          'value': 0.0001},
 'interestRate': None,
 'markPrice': None,
 'nextFundingDatetime': None,
 'nextFundingRate': None,
 'nextFundingTimestamp': None,
 'previousFundingDatetime': '2023-08-12T12:00:00.000Z',
 'previousFundingRate': 0.0001,
 'previousFundingTimestamp': 1691841600000,
 'symbol': 'ETH/USDT:USDT',
 'timestamp': None}
```

Reply:
> hum this timestamp seems to be the "nextFundingTimestamp` (right now are 14h29 UTC time and the next funding it at 20h00) ![image](https://user-images.githubusercontent.com/43336371/260236019-f6492cbf-00b5-459c-b61a-b97b6e46bed1.png)
> 
> ```
> CCXT v4.0.57
> kucoinfutures.fetchFundingRate(ETH/USDT:USDT)
> {'datetime': None,
>  'estimatedSettlePrice': None,
>  'fundingDatetime': '2023-08-12T20:00:00.000Z',
>  'fundingRate': 0.0001,
>  'fundingTimestamp': 1691870400000,
>  'indexPrice': None,
>  'info': {'granularity': 28800000,
>           'predictedValue': 0.0001,
>           'symbol': '.ETHUSDTMFPI8H',
>           'timePoint': 1691841600000,
>           'value': 0.0001},
>  'interestRate': None,
>  'markPrice': None,
>  'nextFundingDatetime': None,
>  'nextFundingRate': None,
>  'nextFundingTimestamp': None,
>  'previousFundingDatetime': '2023-08-12T12:00:00.000Z',
>  'previousFundingRate': 0.0001,
>  'previousFundingTimestamp': 1691841600000,
>  'symbol': 'ETH/USDT:USDT',
>  'timestamp': None}
> ```

so I guess that's what it's supposed to be?